pkgs/applications/networking/instant-messengers/viber/default.nix

```diff
@@ -6,12 +6,12 @@
 
 stdenv.mkDerivation {
   pname = "viber";
-  version = "13.3.1.22";
+  version = "16.1.0.37";
 
   src = fetchurl {
     # Official link: https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb
-    url = "http://web.archive.org/web/20210602004133/https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb";
-    sha256 = "0rs26x0lycavybn6k1hbb5kzms0zzcmxlrmi4g8k7vyafj6s8dqh";
+    url = "https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb";
```

Comment:
```suggestion
    url = "https://web.archive.org/web/20211119123858/https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb";
```
we wabt stable URLs

Reply:
For me web-archive is unbelievably slow, like ~100 KB/sec, while I'm writing this, it's not finished and not to be done very soon.

But yeah, they provide a not reliable source to download.

Reply:
or it's not downloading it at all:

```
[1/0/12 built, 0.0 MiB DL] building viber.deb:   0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
```

Reply:
```
[1/0/12 built, 0.0 MiB DL] building viber.deb:   0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--
error: hash mismatch in fixed-output derivation '/nix/store/rfr4cnb75ayw9a7m7lqgrc07rv6vy9wk-viber.deb.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-hOz+EQc2OOlLTPa2kOefPJMUyWvSvrgqgPgBKjWE3p8=
error: 1 dependencies of derivation '/nix/store/ljn54ibr7kxdk933s5dfhddakd5h42np-viber-16.1.0.37.drv' failed to build
error: 1 dependencies of derivation '/nix/store/v5jkqcvzcamz1yblg9aq56312hr37q2q-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/rjiyrdnvvibwwa1m0c4va7adnrrxq67r-nixos-system-xps-21.11.20211115.931ab05.drv' failed to build
nixos-config on  main [$!+] took 18m31s ❯
```

downloaded in 19 minutes.